update_product_stream/README.md

```diff
@@ -113,8 +125,15 @@ The following requirements must be met on the system where the procedure is bein
    tar -cvzf "csm-${NEW_CSM_RELEASE}.tar.gz" "csm-${NEW_CSM_RELEASE}/"
    ```
 
+1. This step must be performed if an http proxy was set previously.
```

Comment:
I don't think this set of unsets is needed.   From above, it looks like they should already have the patch so there is nothing in this set of steps that goes out to the internet.